src/current/molt/molt-fetch.md

```diff
@@ -369,10 +372,71 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
 `import-only` instructs MOLT Fetch to load the source data in the specified [cloud storage](#cloud-storage) or [local file server](#local-file-server) into the CockroachDB target.
 
 {% include_cached copy-clipboard.html %}
-~~~ 
+~~~ shell
 --mode import-only
 ~~~
 
+#### Fail back to source database
+
+{{site.data.alerts.callout_info}}
+`failback` mode supports MySQL 8.0 and later.
+{{site.data.alerts.end}}
+
+If you encounter issues after moving data to CockroachDB, you can use `failback` to replicate changes on CockroachDB back to the initial source database. In case you need to roll back the migration, this ensures that data is consistent on the source.
```

Comment:
@rohan-joshi this is not the only case where customers fail back right? Sometimes, folks want to write data back just as a backup or if they want to keep their original DB up to date just in case right? Not only during failures.

Reply:
Sometimes, folks want to write data back just as a backup or if they want to keep their original DB up to date just in case right?  -> I don't see that as a use case.

I think the primary use case that ryan highlighted is correct